glide.lock

```diff
@@ -0,0 +1,19 @@
+hash: e51435ca46013a2a19b79acf3103c4e7d7b889d54d0d636fdc670e7f29bcf632
+updated: 2016-02-19T00:10:20.102933642+03:00
+imports:
+- name: github.com/golang/protobuf
```

Comment:
Didn`t notice that. I`ve ran ```glide init``` and it tries to guess what packages are required. I think it can be removed.